libraries/AP_HAL_ChibiOS/hwdef/luminousbee5/hwdef.dat

```diff
@@ -84,13 +95,15 @@ PB15 SPI2_MOSI SPI2
 PC0 VBUS_VALID INPUT PULLDOWN
 
 #BMI088 CS
-PC2 BMI088_ACCEL_CS CS
-PC15 BMI088_GYRO_CS CS
+#PC2 BMI088_ACCEL_CS CS	#opt. second imu
+#PC15 BMI088_GYRO_CS CS	#opt. second imu
 
 PC3 LED_SAFETY OUTPUT
 PC4 SAFETY_IN INPUT PULLDOWN
 PC5 VDD_PERIPH_EN OUTPUT HIGH
-PC7 TIM3_CH2 TIM3 RCININT PULLDOWN LOW # also USART6_RX for serial RC
+PC7 TIM3_CH2 TIM3 RCININT FLOAT LOW # also USART6_RX for serial 
```

Comment:
why change to FLOAT? This could lead to bad detection if a receiver is not connected

Reply:
Mm . I don't know exactly why this was changed from PULLDOWN to FLOAT . 
Also because you had yourself changed it in 5.1 with this commit :
https://github.com/ArduPilot/ardupilot/blob/47da7f5c9b5913ac12f95713c7ff08e7621c330c/libraries/AP_HAL_ChibiOS/hwdef/luminousbee5/hwdef.dat
I will check with my hardware colleagues , but if you tell us that it is better to keep it at PULLDOWN , we will change it right away .

Reply:
@tridge 
Hello , then you are right ! It must have been an oversight going between versions . I will change it